.github/workflows/coding-standards.yml

```diff
@@ -18,9 +18,10 @@ on:
       - '3.[89]'
       - '[4-9].[0-9]'
     paths:
-      # Any change to a PHP or JavaScript file should run checks.
-      - '**.js'
-      - '**.php'
+      # Any change to a relevant PHP or JavaScript file should run checks.
+      - 'src/**.js'
+      - 'src/**.php'
+      - 'tests/**.php'
```

Comment:
Misses files in the root directory, currently the PHP sample configs, webpack.

Which files in particular are you trying to ignore here? 

:1234: This applies to other changes too, so I won't repeat myself.

Reply:
> Misses files in the root directory, currently the PHP sample configs, webpack.

Good catch. PHP files in the root are indeed covered by phpcs so these need re-adding to the path filtering.

The webpack files aren't covered by eslint so they can remain excluded: https://github.com/WordPress/wordpress-develop/blob/cfcb4e9e168b557ea8867ac2b6729c25906a7f54/.eslintignore#L6